torchci/pages/api/classifier/rules.ts

```diff
@@ -179,6 +179,16 @@ export default async function handler(
         pattern: r`undefined reference to .*`,
         priority: 200,
       },
+      {
+        name: "SIG Code",
+        pattern: r`^.*Received Signal: SIG.*`,
+        priority: 150,
+      },
+      {
+        name: "SIGKILL",
+        pattern: r`^.*SIGKILL.*`,
```

Comment:
out of curiosity, why does sigkill get its own rule?

Reply:
It's an experiment. Catching a wider net in case the first one doesn't cut it since we've been getting lots of sigkills lately.